core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/JarResultBuildStep.java

```diff
@@ -1082,7 +1076,7 @@ private void copyCommonContent(FileSystem runnerZipFs, Map<String, List<byte[]>>
             for (TransformedClassesBuildItem.TransformedClass i : transformed) {
                 Path target = runnerZipFs.getPath(i.getFileName());
                 handleParent(runnerZipFs, i.getFileName(), seen);
-                try (final OutputStream out = wrapForJDK8232879(Files.newOutputStream(target, DEFAULT_OPEN_OPTIONS))) {
+                try (final OutputStream out = wrapForJDK8232879(Files.newOutputStream(target))) {
```

Comment:
I kept `wrapForJDK8232879` as it has only been fixed in 11.0.6 so could still be a problem.